packages/core/src/components/region/RegionFilter/RegionFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import { useRef } from 'react'
+
+import {
+  Button as UIButton,
+  Icon as UIIcon,
+  useUI,
+  regionSliderTypes,
+} from '@faststore/ui'
+
+type RegionFilterProps = {
+  label: string
+  icon: {
+    alt: string
+    icon: string
+  }
+}
+
+function RegionFilter({ label, icon: { icon, alt } }: RegionFilterProps) {
```

Comment:
```suggestion
function RegionFilterButton({ label, icon: { icon, alt } }: RegionFilterProps) {
```

suggestion: Specify that this is only the filter button.